.babelrc

```diff
@@ -15,6 +15,9 @@
 	"plugins": [
 		"add-module-exports",
 		"syntax-jsx",
+		[ "transform-builtin-extend", {
```

Comment:
It is possible to enable this plugin only for test env. See: https://babeljs.io/docs/usage/babelrc/.